src/read.c

```diff
@@ -2443,7 +2450,12 @@ static UInt ReadStats (
             SyntaxError("statement expected");
         }
         nr++;
-        MatchSemicolon(follow);
+        if( STATE(Symbol) != S_PRAGMA ){
+            MatchSemicolon(follow);
+        }
+        else {
+            Match(S_PRAGMA, "", 0L );
+        }
```

Comment:
What's the reason for this change?

Reply:
While a pragma is a statement, you have to `Match` the statement end to get the next symbol. However, a pragma does not end with a semicolon, so we cannot get the next symbol by matching semicolon.